README.md

```diff
@@ -171,5 +174,59 @@ from the current profile whenever it is reset by calling `setProfile` or `resetP
 You should call `setPushToken` again after resetting the currently tracked profile 
 to explicitly to associate the device token to the new profile.
 
+### Deep linking in push notification 
+
+In order to set up a push notification to deep link into your apps, there are broadly three steps - 
+1. Add intent filters for incoming links. 
+2. Read the data from the incoming links and route to the appropriate views.
+3. Test your deep links.
+
+#### Step 1: Add intent filters for incoming links
+
+1. Add the below XML into your `AndroidManifest.xml` 
+2. Replace the scheme to match your app's scheme. Essentially, you would replace example with whatever scheme you want your app to use. We recommend this be unique to your app.
+
+
+```xml
+    <intent-filter android:label="@string/filter_view_example_gizmos">
+        <action android:name="android.intent.action.VIEW" />
+        <category android:name="android.intent.category.DEFAULT" />
+        <category android:name="android.intent.category.BROWSABLE" />
+        <!-- Accepts URIs that begin with "example://” -->
+        <data android:scheme="example"/>
```

Comment:
We may want to provide `host` here too to give a slightly more detailed description. Like what we saw in the FCM docs around this. scheme + host with a comment that outlines how that forms a URI